app/views/components/add_save_button.scala.html

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  *@
 
+@import controllers.util.{Add, SaveAndContinue}
+
 @(
     saveKey: String = "site.save_and_continue",
     addKey: String = "site.add"
 )(implicit messages: Messages)
 
 <div class="section align='right'">
-    <button id="add" name="action" class="button" value="@messages(addKey)">@messages(addKey)</button>
-    <button id="submit" name="action" class="button" value="@messages(saveKey)">@messages(saveKey)</button>
+    <button id="add" class="button" name="@Add.toString" value="">@messages(addKey)</button>
```

Comment:
You can remove `value=""`. You don't use it.

Reply:
You're right